cmd/odf/maintenance/start.go

```diff
@@ -0,0 +1,22 @@
+package maintenance
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/maintenance"
+	"github.com/spf13/cobra"
+)
+
+var startMaintenanceCmd = &cobra.Command{
+	Use:     "start",
+	Short:   "Start a maintenance deployment with an optional alternative ceph container image",
+	Args:    cobra.ExactArgs(1),
+	Example: " odf maintenance start <deployment_name>",
+	Run: func(cmd *cobra.Command, args []string) {
+		alternateImage := cmd.Flag("alternate-image").Value.String()
+		maintenance.StartMaintenance(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, args[0], alternateImage)
```

Comment:
is this function already implemented and tested, plus we do have the ci for this?

Reply:
Yes, this is already implemented and tested, and have CI upstream as of now we don't have downstream CI with openshift due to a lot of overhead complexities.